### PR TITLE
feat: Installing Vale CLI with the RedHat package

### PR DESCRIPTION
**Pull request by themr0c**
feat: Installing Vale CLI with the RedHat package. Explain exclusively using the package, remove references to methods using the Git repository.
fix: search (was broken due to obsolets files in the supplemental-ui, that now belong to the search extension).

fixes: https://github.com/redhat-documentation/vale-at-red-hat/issues/245
fixes: https://github.com/redhat-documentation/vale-at-red-hat/issues/244

Screenshot from 2022-08-23 15-28-56

### CODE REVIEW

**Comment by aireilly** [comment]
LGTM

**Review by rolfedh — COMMENTED** [review]
The content looks good. @tmalove and @michelle-purcell have been working on related updates to the content. Please coordinate with them before merging this change.

File: modules/user-guide/pages/installing-vale-cli.adoc [hunk 1]
```diff
@@ -1,49 +1,60 @@
...
-= Installing and configuring Vale by cloning the *{repository-name}* repo
-
-To use `vale` CLI on any project in your local environment, install the `vale` CLI and register the *{repository-name}* Vale configuration as the default.
-
-NOTE: Alternatively, if you do not intend to contribute to the *{repository-name}* project, you can configure the `RedHat` Vale style by downloading the latest release from GitHub. See xref:installing-vale-from-zip.adoc[]
-
-.Prerequisites
+// End of metadata for Modular Docs
+[id="proc_using-vale-cli_{context}"]
+= Installing Vale with the `RedHat` package
```
Comment (michelle-purcell) [inline modules/user-guide/pages/installing-vale-cli.adoc:12]: At this point, will the user be familiar with the concept of 'packages'. If not, perhaps the following might increase findability:

Suggestion: 'Installing Vale with the preconfigured Red Hat style ruleset package'

File: modules/user-guide/pages/installing-vale-cli.adoc [hunk 2]
```diff
@@ -1,49 +1,60 @@
...
-. Install the Vale command-line tool on your workstation. See link:https://docs.errata.ai/vale/install[Installing Vale].
+. Install the Vale command-line tool on your workstation.
+See link:https://vale.sh/docs/vale-cli/installation/[Installing Vale].
 +
-NOTE: On the Vale site, click the tab for your operating system.
+[NOTE]
+On the Vale site, click the tab for your operating system.
 
-. Clone the link:{repository-url}[{repository-name} repository] in your local environment.
+. Register the `RedHat` Vale package in the `.vale.ini` Vale configuration file. Save the file in your `$HOME` directory:
```
Comment (michelle-purcell) [inline modules/user-guide/pages/installing-vale-cli.adoc:24]: When I tested this procedure,I already had a preconfigured .vale.ini in my $home folder. It got me thinking that another use case is:  Switching to packages when you have already been using Vale for some time.

**Comment by michelle-purcell** [comment]
@Fabrice. Looks great. Thank you for simplifying the ruleset configuration process part of the installation. 

In a future iteration, one suggestion might be to divide the content into sub-sections, something like:

### Installing the Vale client

### Installing and configuring the Red Hat style ruleset for Vale:

#### Installing the vale-at-red-hat ruleset package 
#### Installing the vale-at-red-hat ruleset repo for contributing

@tmalove and I can help with this effort and discuss with you further. Thanks :-)

**Review by michelle-purcell — APPROVED** [review]

**Comment by themr0c** [comment]
To keep things lean and clean, I believe the contributing section should go in the contributing guide, not in the user guide.

**Pull request merged**